src/main/java/org/checkerframework/specimin/UnsolvedMethod.java

```diff
@@ -27,6 +27,12 @@ public class UnsolvedMethod {
   /** This field is set to true if this method is a static method */
   private boolean isStatic = false;
 
+  /**
+   * Indicates whether this instance of UnsolvedMethod represents just a method signature without a
+   * body.
+   */
+  public boolean isJustMethodSignature = false;
```

Comment:
Why is this field public and mutable? That combination should not be created without a very good reason. I'd prefer a private final field, if at all possible.